docs/decisions/0038-use-entryId-for-bibentries.md

```diff
@@ -22,9 +22,10 @@ This, however, is not useful in the UI, where equal entries are not the same ent
 
 ## Considered Options
 
+* Use `BibEntry.getId` for indexing `BibEntry`
```

Comment:
Do not rename options, if you add a new option :)

I re-added it in a a1461e59c3

Maybe add a sentence, why this is bette than identityHashCode. Is it faster? Is it more close to BibEntry object?